README.md

```diff
@@ -52,6 +54,45 @@ then you can process them with:
 bin/process-schemas-for-redoc
 ```
 
+## Schema transformations
+
+The transformer applies several optimizations to make the OpenAPI schemas
+more suitable for Redoc documentation:
+
+### Optimization for Redoc
+
+- **Sorting**: All API paths are sorted alphabetically for consistent ordering.
+  See [PathSorter.test.js](src/__tests__/PathSorter.test.js).
+- **Summary generation**: Automatically generates endpoint summaries based on
+  their tags for better readability. See [SummaryGenerator.test.js](src/__tests__/SummaryGenerator.test.js).
```

Comment:
I'm not sure what the line returns are for. Why isn't this one line? Does it look better as two? This pattern repeats throughout the readme.

Reply:
That's was just recommended line-length fix auto-correction.